.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Publish
+
+on:
+  workflow_dispatch:
+
+env:
+  node-version: lts/*
+
+jobs:
+  build:
+    name: Build and publish the latest Rainbow Token List
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Use Node ${{ env.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.node-version }}
+          cache: yarn
+
+      - run: yarn
+      - run: yarn lint
+      - run: yarn test --ci --coverage --maxWorkers=2
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - run: aws s3 cp output/rainbow-token-list.json s3://rainbow-metadata-production/token-list/rainbow-token-list.json
+      - run: aws cloudfront create-invalidation --distribution-id "metadata-productionS3Origin" --paths "/token-list"
```

Comment:
Were you able to test this action? I imagine we want to confirm it works before merging it down -> running manual push -> creating new PR to fix any issues?